deployment/nether-deploy-quickstart.json

```diff
@@ -1,93 +1,256 @@
 {
-    /*
-    * This is the "master" template used to provision the Azure resources for Nether.
-    *
-    * This leverages linked templates which requires that the templates it links to are available via a URL. By default, these 
-    * will be pulled from the same location as this template. But this can be overridden by specifying a location via the pull 
-    * them from by specifying the templateBaseURL parameter.
-    * 
-    * For more information, please refer to the Nether repository at: https://github.com/MicrosoftDX/nether/tree/master/deployment
-    */
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "initialNetherAdministratorPassword": {
-            "type": "securestring",
-            "metadata": {
-                "description": "The initial password for the netheradmin user, sql server user, ..."
-            }
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "initialNetherAdministratorPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The initial password for the netheradmin user, sql server user, ..."
+      }
+    },
+    "templateBaseURL": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/MicrosoftDX/nether/master/deployment/",
+      "metadata": {
+        "description": "The base location for all linked templates"
+      }
+    },
+    "WebHostingPlan": {
```

Comment:
We need the quick-start-template to be as easy to use as possible, therefor we should not ask the user about things like web hosting plans, etc. it should all be our defaults that apply